paddle/fluid/pybind/imperative.cc

```diff
@@ -1023,6 +1030,7 @@ void BindImperative(py::module *m_ptr) {
               y = x.cuda(1)
               print(y.place)        # CUDAPlace(1)
        )DOC")
+      .def("copy_", &imperative::VarBase::CopyFrom)
```

Comment:
why not write `__deepcopy__` here directly?

Reply:
named `copy_`, is it will show to users?

Reply:
Want to expose it to the user, user can use tensor1.copy_(tensor2) to copy another tensor to this tensor inplacely.

Reply:
> why not write `__deepcopy__` here directly?

__deepcopy__(self, memo)  memo will record this copy by memo[id(self]=copy_var, need to get id(self),  if deecopy twice, a same tensor will return.  So place it in python. 
There's no other difference between the two implementations.